tests/pipelines/test_pipelines_image_text_to_text.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
Can we add pipeline mapping in all VLMs so that they are tested with image-text-to-text pipeline? We have to make sure the pipeline works with all VLMs, and some of the models are very peculiar. For ex in BLIP-2 we have

https://github.com/huggingface/transformers/blob/293e6271c69a48b6a66e68978dd3d37601c04c63/tests/models/blip_2/test_modeling_blip_2.py#L703-L708